src/screens/Loading.js

```diff
@@ -63,25 +64,26 @@ export default class Loading extends React.PureComponent {
 
   async migrateAccounts() {
     const oldAccounts_v1 = await loadAccounts(1);
-    // v2 (up to v2.2.2) are only ethereum accounts 
+    // get a map from old accounts
+    // v2 accounts (up to v2.2.2) are only ethereum accounts 
     // with now deprectaded `chainId` and `networkType: 'ethereum'` properties
     // networkKey property is missing since it was introduced in v3.
     const oldAccounts_v2 = await loadAccounts(2);
-    const oldAccounts = [...oldAccounts_v1, ...oldAccounts_v2]
-    const accounts = oldAccounts.map(a => {
+    const oldAccounts = [...oldAccounts_v1, ...oldAccounts_v2];
```

Comment:
I don't know it even works without `entries`

Reply:
Not sure what you mean. `loadAccounts` retrurns a map, using the spread is allowed and documented https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map#Relation_with_Array_objects

Reply:
Thanks, I thought I can only use it with `[...oldAccounts_v1.entries()]`